starter-code/css/main.css

```diff
@@ -0,0 +1,85 @@
+
+
+body {
+  font-family: arial, verdana;
+  font-variant: small-caps;
+  padding:20px;
+  border-left: 4px solid grey;
+  margin-left: 60px;
+
+}
+a {
+  text-decoration: none;
+  color: #ffd700;
+}
+h1 {
+  font-size: 50px;
+  font-weight: 100;
+  color: #ffd700;
+  text-decoration: none;
+}
+nav {
+  background-color: transparent;
+  height: 100%;
+  height: 60px;
```

Comment:
You have the same property `height` being set twice here. That means the one the browser will actually apply is the last one since it cascades down this file. Therefore line 23 is unnecessary.